config/environment.js

```diff
@@ -21,7 +21,7 @@ module.exports = function(environment) {
     'labs-search': {
       host: 'https://search-api-production.herokuapp.com',
       route: 'search',
-      helpers: ['geosearch', 'waterfront-park-name'],
+      helpers: ['geosearch-v2', 'waterfront-park-name'],
```

Comment:
It looks like this `helpers` property is also set down on line 92 for when environment is "production". I would prefer you just delete that line because IMO we should only set `helpers` once because it isn't environment-dependent.

Reply:
oof good catch.

Reply:
@TylerMatteo  fixed

Reply:
Thanks. Seems a little odd to me that this app doesn't seem to point to staging Search API in lower environments but I'm not concerned with addressing that here.